portal-frontend/src/app/shared/no-data/no-data.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="no-data">
-  <div class="text">No Data</div>
+  <div class="text">{{ text || 'No Data' }}</div>
```

Comment:
Since the logic for showing No Data is often more complicated than just a piece of text existing, such as number of flles or other conditions, I dont think moving the text inside the component is a good idea. It will create multiple patterns of using this component instead of a single way.

Reply:
I think I already refactored this and not using it anywhere. I'll revert changes